### PR TITLE
Add Maggie Play For Pat to Review

### PR DESCRIPTION
**Pull request by mgullon**
Hi Pat, 

Please review.

### CODE REVIEW

File: _posts/0000-05-02-mgullon.md [hunk 1]
```diff
@@ -8,3 +8,5 @@ title: "Welcome to our slide deck!"
 
 Maggie test2
 Hello
+
```
Comment (pbreslin) [inline _posts/0000-05-02-mgullon.md:11]: Hello back!

File: _posts/0000-05-02-mgullon.md [hunk 2]
```diff
@@ -8,3 +8,5 @@ title: "Welcome to our slide deck!"
...
+Maggie Play
```
Comment (pbreslin) [inline _posts/0000-05-02-mgullon.md:12]: More bad

**Review by pbreslin — CHANGES_REQUESTED** [review]

File: _posts/0000-05-02-mgullon.md (hunk 2 above)
Comment (mgullon) [reply _posts/0000-05-02-mgullon.md:12]: Please let me in!

**Comment by pbreslin** [comment]
OK!

**Review by pbreslin — APPROVED** [review]
It's great now!

**Pull request merged**